src/main/java/org/pentaho/di/trans/dataservice/serialization/DataServiceMetaStoreUtil.java

```diff
@@ -158,6 +159,16 @@ public DataServiceMeta getDataService( String serviceName, TransMeta serviceTran
         } catch ( Exception e ) {
           exceptionHandler.apply( e );
         }
+      } else {
+        try {
+          if ( SecurityHelper.getInstance().runAsSystem(
```

Comment:
There's a NPE that occurs here in SecurityHelper.createAuthentication() when in spoon, not connected to a repo.